dev/breeze/src/airflow_breeze/commands/testing_commands.py

```diff
@@ -679,6 +679,11 @@ def _run_test_command(
     fix_ownership_using_docker()
     cleanup_python_generated_files()
     perform_environment_checks()
+    if pydantic:
```

Comment:
```suggestion
    if pydantic != "v2" or downgrade_sqlalchemy or downgrade_pendulum or remove_arm_packages:
```

Reply:
I think this might **potentially** happen in those cases too.

Reply:
(and pydantic == 'v2' is default)

Reply:
Unlikely happen in other cases rather than pydantic. This happen because Weaviate depend on pydantic, and all tests just skipped.

In case of pendulum and sqlalchemy cases it should works with both versions.

boto3 related to mostly amazon provider (if other even related on them), and there hundreds tests which do not required aiobotocore

Reply:
So maybe `pydantic != "v2"` would be enough now, and we might extend when we found if it happen in other cases

Reply:
yeah. Would be ok now.

Reply:
Add `pydantic != "v2"` instead of every time